shared/actions/kbfs/platform.native.js

```diff
@@ -2,6 +2,6 @@
 
 import type {AsyncAction} from '../../constants/types/flux'
 
-export function platformOpenInKBFS (path: string = ''): AsyncAction {
+export function openInKBFS (path: string = ''): AsyncAction {
```

Comment:
this is the original bug: https://github.com/keybase/client/blob/master/shared/actions/kbfs/index.js#L7 it expects an export called 'openInKBFS' which it renames to platformOpenInKBFS (since its delegating this out)